.travisrc.yml

```diff
@@ -0,0 +1,22 @@
+language: node_js
+
+node_js:
+  - "8"
+  - "10"
+  - "11"
```

Comment:
Let's switch to `12` which is released.

Reply:
so 10, 11, 12? or 8, 10, 12?

Reply:
8, 10, 12, aka the "official releases". We only do odd-numbered unstable releases when they're out and close enough to official even status to get a heads up on...